retriever/lib/engine.py

```diff
@@ -35,6 +35,8 @@ class Engine(object):
     _cursor = None
     datatypes = []
     required_opts = []
+    pgpass_lookup = {}
```

Comment:
we should not have these two changes here in the main engine. Since they have nothing to do with other engines, we should keep these changes to the specific engines.